src/main/java/org/spongepowered/mod/service/scheduler/SyncScheduler.java

```diff
@@ -41,7 +41,7 @@
 /**
  * <p>Internal implementation of the SynchronousScheduler interface.</p>
  *
- * @see {@link org.spongepowered.api.service.scheduler.SynchronousScheduler}
+ * See {@link org.spongepowered.api.service.scheduler.SynchronousScheduler}
```

Comment:
I'll revert it and throw it in another PR, thanks for providing the correct syntax.